framework/src/utils/MooseCoordTransform.C

```diff
@@ -0,0 +1,276 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MooseCoordTransform.h"
+#include "InputParameters.h"
+#include "MultiMooseEnum.h"
+#include "MooseEnum.h"
+#include "MooseMesh.h"
+
+#include "libmesh/tensor_value.h"
+#include "libmesh/point.h"
+
+using namespace libMesh;
+
+MooseCoordTransform::MooseCoordTransform() {}
```

Comment:
cant you say = default in the header?

Reply:
If you do, then you have to make sure you include the headers of things held by `unique_ptrs`. That's the only drawback to `= default;`